src/axom/slam/tests/slam_relation_DynamicVariable.cpp

```diff
@@ -213,10 +213,7 @@ TEST(slam_relation_dynamic_variable, iterate_relation)
 int main(int argc, char* argv[])
 {
   ::testing::InitGoogleTest(&argc, argv);
-
-  // create & initialize test logger. finalized when exiting main scope
-  axom::slic::SimpleLogger logger;
-  axom::slic::setLoggingMsgLevel(axom::slic::message::Info);
+  axom::slic::SimpleLogger logger(axom::slic::message::Info);
```

Comment:
Why would a test not want debug messages?

Reply:
Not sure -- maybe it was too verbose and hiding what we really wanted to see? I didn't look into it in this PR, I just followed what was already there.